### ISSUE TITLE
Restructure generator.py according to the utilities

### ISSUE THREAD

**Issue by ashwini0529** [pytorn/hackr#8]
As of now, there are multiple functionalities implemented in a single file. Most of the generator functions are `Faker` oriented. Maybe a separate file handles such functions.

**Comment by theycallmemac** [comment]
Just had a look at this. You want the functions using `Faker` to be separated from the few that don't?

**Comment by ashwini0529** [comment]
Yeah basically the purpose of `generator` is to generate data. Now that maybe using `Faker`, `built-in` functions, or any other way. `generator.py` should be in such a way, that it should not contain much of a repeated code, or functions.

**Comment by theycallmemac** [comment]
So rather than having separate files handling such functions as you've said above, this is just a refactoring issue?

**Comment by ashwini0529** [comment]
Yeah kind of refactoring issue.

**Comment by ashwini0529** [comment]
Hey @theycallmemac , can you please check a comment that I made on your PR? 
That would be ready to merge then. 🎉

**Comment by theycallmemac** [comment]
Isn't it merged already?
